data/human/news.txt

```diff
@@ -1990,6 +2093,75 @@ news "conspiracy theorist"
 
 
 
+news "players crew member"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	to show
+		"flagship crew" > 30
+	name
+		word
+			"Some crew member"
+	message
+		word
+			`"`
+		word
+			"I'm serving onboard the '<ship>', but I've never met the captain. Can you imagine that?"
+			"Hey captain, good to see you here. I'm looking forward to the next trip (and I'm looking forward to my 100 credits a day)."
+			"You look like a poor excuse for a captain. I'm serving onboard the '<ship>' and I'm sure that my captain is far better dressed than you."
+		word
+			`"`
+
+
+
+news "sticker on ship"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Sticker on ship"
+	message
+		word
+			"I brake for asteroid snails."
+			"Student pilot."
+			"Babe onboard."
+			"There is no place like [insert homeplanet here]."
+
+
+
+news "label on ship"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Label on ship"
+			"Warning label on ship"
+	message
+		word
+			"Angles morts - Blind spots from cockpit."
+			"Stay clear of ramscoop intake."
+			"Stay out of thruster exhaust stream."
+			"Stay away form thruster outlets during takeoff."
+			"Door opens outwards."
+			"Remove red ribbons before flight."
+			"Nuclear reactor compartiment, NEA certification required for servicing this part of the ship."
```

Comment:
I feel like this could almost be combined with the previous one.

Reply:
NEA: https://www.oecd-nea.org/
But they are apparently not well-known, so I removed the NEA abbreviation.

Reply:
> I feel like this could almost be combined with the previous one.

Almost, but I feel that `label` has a bit of a more formal meaning (added by the manufacturer of a ship, or by some official organization), while bumper sticker is more something unofficial (a joke on a sticker, put on a bumper or rear window).

Reply:
The meaning of NEA can be guessed from context and I felt it was a nice touch

Reply:
Re-added the abbreviation for the Northern Evil Authors.